test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/WarmupFunctionEndToEndTests.cs

```diff
@@ -60,7 +60,16 @@ public async Task Proxy_Admin_Override_Fails()
             HttpResponseMessage response = await _fixture.HttpClient.GetAsync("/admin/123");
 
             string content = await response.Content.ReadAsStringAsync();
-            Assert.True(response.StatusCode.ToString("D") == "404", "/admin/* endpoints cannot be overriden by proxies.");
+            Assert.False(response.Headers.Contains("myversion"), "/admin/* endpoints cannot be overriden by proxies.");
+        }
+
+        [Fact]
+        public async Task FunctionRoutes_Admin_Override_Fails()
+        {
+            HttpResponseMessage response = await _fixture.HttpClient.GetAsync("/admin/host/status");
+
+            string content = await response.Content.ReadAsStringAsync();
+            Assert.False(content == "Pong", "/admin/* endpoints cannot be overriden by function routes.");
```

Comment:
Can you add a positive check here like you did in the test above? E.g. look for myversion, or preferably verify that the body contains status details